default/python/universe_generation/galaxy.py

```diff
@@ -45,32 +307,10 @@ def calc_universe_width(shape, size):
     return width
 
 
-def get_systems_within_jumps(origin_system, jumps):
-    """
-    Returns all systems within jumps jumps of system origin_system (including origin_system).
-    If jumps is 0, return list that only contains system origin_system.
-    If jumps is negative, return empty list.
-    """
-    if jumps < 0:
-        return []
-    matching_systems = [origin_system]
-    next_origin_systems = [origin_system]
-    while jumps > 0:
-        origin_systems = list(next_origin_systems)
-        next_origin_systems = []
-        for system in origin_systems:
-            neighbor_systems = [s for s in fo.sys_get_starlanes(system) if s not in matching_systems]
-            next_origin_systems.extend(neighbor_systems)
-            matching_systems.extend(neighbor_systems)
-        jumps -= 1
-    return matching_systems
```

Comment:
`galaxy.py` is a misnomer.  It should more properly by named `galaxy_morphology.py`.  `get_systems_within_jumps()` does not belong in `galaxy.py`.

All of the functions/classes in `galaxy.py`, are directly or indirectly called by `calc_star_system_positions()` to generate position data.  Except `get_systems_within_jumps()`.

None of the functions know anything about starlanes or deal with starlanes.    Except `get_systems_within_jumps()`.

`get_systems_within_jumps()` doesn't belong in this file.  The only place that it is called is in `empires.py`.  There are similar functions in `empires.py` which also call `fo.` to access starlane information, such as `systems_min_jumps_from()`.  I moved the definition of `get_systems_within_jumps()` to just before it is used. 

I could:
1. Do nothing.  Leave `get_systems_within_jumps()` in `empires.py`.  I like this option.
2. Restore `get_systems_within_jumps()` to `galaxy.py`. 
3. Rename `galaxy.py` to `galaxy_morphology.py`.  Create `galaxy_topology.py` and move `get_systems_within_jumps()` into it.  

I'll do whichever option you prefer.